src/library/export/exportrequest.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <QSet>
+
+#include "library/trackset/crate/crateid.h"
+
+namespace mixxx {
+
+/// The ExportRequest struct is the base class for all types of request to export
+/// the Mixxx library to some external location.  An export to a specific kind of
+/// external location is likely to need additional information to be specified in
+/// the request, which can be achieved by sub-classing this struct.
+struct ExportRequest {
```

Comment:
Without a virtual destructor derived classes will cause memory leaks when deleted by a pointer to the base struct/class.

Reply:
Added.